src/Components/JobCard/JobCard.js

```diff
@@ -1,13 +1,283 @@
+              <JobCompanyLocation>
+                서울
+                <span>.</span>
+                한국
+              </JobCompanyLocation>
+              <Reward>채용보상금 1,000,000원</Reward>
+            </Contents>
+          </a>
+        </div>
+      </JonCardList>
+      <JonCardList>
+        <div>
+          <a href="#">
+            <header>
+              <ButtonTop type="button">
+                <FavoriteIcon
+                  style={{
+                    marginRight: "5px",
+                    color: "hsla(0,0%,100%,.3)",
+                    fontSize: "15px",
+                  }}
+                />
+                153
+              </ButtonTop>
+              <ButtonBottom type="button">
+                <span>AI 분석</span>
+              </ButtonBottom>
+            </header>
+            <Contents>
+              <JobCardPosition>Web, Mobile Web개발자 (신입)</JobCardPosition>
+              <JobCardCompanyName>뉴지스탁</JobCardCompanyName>
+              <JobCompanyLocation>
+                서울
+                <span>.</span>
+                한국
+              </JobCompanyLocation>
+              <Reward>채용보상금 1,000,000원</Reward>
+            </Contents>
+          </a>
+        </div>
+      </JonCardList>
+      <JonCardList>
+        <div>
+          <a href="#">
+            <header>
+              <ButtonTop type="button">
+                <FavoriteIcon
+                  style={{
+                    marginRight: "5px",
+                    color: "hsla(0,0%,100%,.3)",
+                    fontSize: "15px",
+                  }}
+                />
+                153
+              </ButtonTop>
+              <ButtonBottom type="button">
+                <span>AI 분석</span>
+              </ButtonBottom>
+            </header>
+            <Contents>
+              <JobCardPosition>Web, Mobile Web개발자 (신입)</JobCardPosition>
+              <JobCardCompanyName>뉴지스탁</JobCardCompanyName>
+              <JobCompanyLocation>
+                서울
+                <span>.</span>
+                한국
+              </JobCompanyLocation>
+              <Reward>채용보상금 1,000,000원</Reward>
+            </Contents>
+          </a>
+        </div>
+      </JonCardList>
+      <JonCardList>
+        <div>
+          <a href="#">
+            <header>
+              <ButtonTop type="button">
+                <FavoriteIcon
+                  style={{
+                    marginRight: "5px",
+                    color: "hsla(0,0%,100%,.3)",
+                    fontSize: "15px",
+                  }}
+                />
+                153
+              </ButtonTop>
+              <ButtonBottom type="button">
+                <span>AI 분석</span>
+              </ButtonBottom>
+            </header>
+            <Contents>
+              <JobCardPosition>Web, Mobile Web개발자 (신입)</JobCardPosition>
+              <JobCardCompanyName>뉴지스탁</JobCardCompanyName>
+              <JobCompanyLocation>
```

Comment:
```suggestion
              <Position>Web, Mobile Web개발자 (신입)</CardPosition>
              <CompanyName>뉴지스탁</CompanyName>
              <CompanyLocation>
```
이미 job card 하위에 있는 내용들이기 때문에 컴포넌트 명이 길어질 필요가 없겠네요